code/modules/mob/mob_defines.dm

```diff
@@ -110,3 +110,8 @@
 	var/datum/click_intercept
 
 	var/registered_z
+//sunset start
+	var/adminfrozen = 0 //handle for someone's sleeping time from before they got frozen
```

Comment:
You can just move this to `sunset/code/modules/mob/mob_defines.dm `

Reply:
done